hijack/models.py

```diff
@@ -1 +0,0 @@
-"""Just an empty models file to let the testrunner recognize this as app."""
```

Comment:
this file could be fully removed as there is no models in use

Reply:
The file was delete, I don't see it locally strange.